src/components/BottomSheetScrollView.tsx

```diff
@@ -30,6 +30,11 @@ function BottomSheetScrollView({ handleContentLayout, containerStyle, testId, ch
     }
   }
 
+  // Dismiss keyboard on mount
+  useEffect(() => {
+    Keyboard.dismiss()
```

Comment:
Just wondering, will this have conflicts if we want to implement some experience where the keyboard should automatically appear because there is some input field with auto focus in the bottom sheet?

Shouldn't we try to dismiss the keyboard in the specific action that triggers the dapp open flow?

Reply:
In general I think we should avoid any input field on the bottom sheet. This is similar to #3625 where the keyboard dismisses when the bottom sheet becomes visible. I haven't seen any instances where the `useEffect` is not triggered and the keyboard persists in testing.